src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -803,30 +803,38 @@ g.test('depth_stencil_attachment')
 g.test('depth_stencil_attachment,depth_clear_value')
   .desc(
     `
-  Test that depthClearValue is invalid if the value is out of the range(0.0 and 1.0) when
+  Test that depthClearValue is invalid if the value is out of the range(0.0 and 1.0) only when
   depthLoadOp is 'clear'.
   `
   )
-  .paramsSimple([
-    { depthClearValue: -1.0 },
-    { depthClearValue: 0.0 },
-    { depthClearValue: 0.5 },
-    { depthClearValue: 1.0 },
-    { depthClearValue: 1.5 },
-  ])
+  .params(u =>
+    u
+      .combine('depthLoadOp', ['load', 'clear'] as const)
```

Comment:
re: https://github.com/gpuweb/cts/pull/1640#discussion_r924015954

> It seems to me that `depth_stencil_attachment` test case can also check the case that `depthLoadOp` is `undefined`. Do you mean that we need to test it in `depth_stencil_attachment,depth_clear_value` again?
> 
> Please correct me if `depth_stencil_attachment` doesn't test the case that you want to test.

`depthLoadOp` has three possible values: `'load'`, `'clear'`, and `undefined`. I want to test that depthClearValue isn't validated when `depthLoadOp` is `undefined`. This is only possible when there is no depth attachment channel, so I think the easiest would be to write the format below ([1]) as `depthLoadOp === undefined ? 'stencil8' : 'depth24plus-stencil8'`.

I don't think this particular thing is covered by other tests but please correct me if I'm wrong.

Reply:
Thank you for explaining it. Yes, it looks like there is no existing test that covers depthClearValue when depthLoadOp is undefined. Let me add `undefined` to `.combine('depthLoadOp', ['load', 'clear'] as const)`.